apps/web/public/static/locales/en/common.json

```diff
@@ -2496,7 +2496,9 @@
   "USER_PENDING_MEMBER_OF_THE_ORG": "User is a pending member of the organization",
   "USER_ALREADY_INVITED_OR_MEMBER": "User is already invited or a member",
   "USER_MEMBER_OF_OTHER_ORGANIZATION": "User is member of an organization that this team is not a part of.",
+  "skip_writing_to_calendar": "Do not write to the ICS feed",
   "rescheduling_not_possible": "Rescheduling is not possible as the event has expired",
   "event_expired": "This event is expired",
+  "skip_writing_to_calendar_note": "Note: If your ICS link is read-only (e.g., Proton Calendar), check the box above to avoid errors.",
```

Comment:
> That is perfect; I can see from the update you mentioned "read-only," which specifies the Proton Mail that supports read-only ICS URLs. Since you are sending to email, the Proton Mail will still send the calendar update and will appear on the Proton Calendar. Which I explained in detail here: https://github.com/calcom/cal.com/issues/14300#issuecomment-2167174611

We should also add something along the lines that 'You would have to manually make changes to your calendar' because in case of rescheduling or cancelling, it won't delete or update the existing link. It would only create new events when sent through mail. Lmk what you think